client/src/views/WeatherPage.vue

```diff
@@ -1,168 +1,38 @@
 <template>
-  <div id="app">
-    <main>
-      <div class="search-box">
-        <input
-          type="text"
-          class="search-bar"
-          placeholder="Search..."
-          v-model="query"
-          @keyup.enter="fetchWeather"
-        />
-      </div>
-      <div class="weather-wrap" v-if="typeof weather.main != 'undefined'">
-        <div class="location-box">
-          <div class="location">
-            {{ weather.name }}, {{ weather.sys.state }}
-            {{ weather.sys.country }}
-          </div>
-          <div class="date">{{ dateBuilder() }}</div>
-          <div class="weather-box">
-            <div class="temp">{{ Math.round(weather.main.temp) }} °F</div>
-            <div class="weather">{{ weather }}</div>
+  <div>
+      <div v-for="(weather, index) in weatherList" :key="weather.id">
+          <div v-if="index % 8  === 0">
+              in {{( index * 3)}} hours, it will be
+              {{weather.main.temp}} °F
           </div>
-        </div>
       </div>
-    </main>
   </div>
+  
 </template>
 
 <script>
 export default {
   data() {
-    return {
-      api_key: "8b81c25b4594c01f9e7d4caec4793f7a",
-      url_base: "https://api.openweathermap.org/data/2.5/",
-      query: "",
-      weather: {},
-    };
+      return {
+          api_key: '8b81c25b4594c01f9e7d4caec4793f7a',
+          url_base: 'https://api.openweathermap.org/data/2.5/',
+          lat: "-34.0522",
```

Comment:
it seems as though the lat and lon remain constants. For the purpose of using this component in the greater context of the project, can we turn these into props? [Vue Props](https://vuejs.org/guide/components/props.html)

Reply:
They should be pulling the Lat and Lng values from storage.

They could be passed down to the component as props, but they can also pull directly from storage like most of the other components have done.

Reply:
for this type of component that provides ancillary functionality to a larger context like in the case of adding it to a whole search interface, it should be kept as prop. Otherwise it requires the person implementing the component having to go in a re-write the component to be compatible with the context in which they are trying to use it in. IE /getmethere utilizes query params to flexibly pass down coordinates. If the forecast component has props than I can just pass them down.